.travis.yml

```diff
@@ -6,6 +6,7 @@ go:
 install:
   - make dependencies
   - test -z "$(go fmt ./...)"
+  - test -z "$(golint -min_confidence=1.0 ./modules/host)"
```

Comment:
just use a separate variable (or no variable at all) instead of pkgs. It may be a while before all packages pass golint.